msal/src/main/java/com/microsoft/identity/client/AccountAdapter.java

```diff
@@ -138,4 +138,17 @@ static AccountRecord getAccountInternal(@NonNull final String clientId,
 
         return accountToReturn;
     }
+
+    @Nullable
+    public static String getRealm(@NonNull IAccount account) {
```

Comment:
I don't think we should make this public API - can we move this class down into the `internal` pkg?